python/desc/skycatalogs/utils/sed_tools.py

```diff
@@ -0,0 +1,230 @@
+import os
+import re
+from astropy import units as u
+from astropy.cosmology import FlatLambdaCDM
+import h5py
+import pandas as pd
+
+import numpy as np
+import numpy.ma as ma
+from numpy.random import default_rng
+from dust_extinction.parameter_averages import F19
+import galsim
+
+__all__ = ['ObservedSedFactory', 'Extinguisher', 'AB_mag', 'get_star_sed_path']
+class ObservedSedFactory:
+    _clight = 3e8  # m/s
+    _to_W_per_Hz = 4.4659e13 # conversion of cosmoDC2 tophat Lnu values to W/Hz
```

Comment:
Maybe add a link to the schema in gcr-catalogs where this conversion is found: https://github.com/LSSTDESC/gcr-catalogs/blob/master/GCRCatalogs/SCHEMA.md

Reply:
Fixed in next push